contents/product-analytics.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Product analytics
+subtitle: Funnels, path analysis, trend charts, retention graphs, and more
+featuredImage: ./images/products/product-analytics.png
+template: product
+productFeatures: [
+    {
+        icon: 'EventAutocapture',
+        title: 'Autocapture',
+        description:
+            'Add PostHog.js to your website or web app to track all event data and retroactively define events',
+    },
+    {
+        icon: 'Cohorts',
+        title: 'Dataviz',
```

Comment:
is this a common term in the US?

Reply:
Yes but will make it _data visualization_ just so it's super clear across the pond